packages/xdl/src/ProjectSettings.ts

```diff
@@ -131,6 +131,30 @@ export function dotExpoProjectDirectory(projectRoot: string): string {
   }
 
   fs.mkdirpSync(dirPath);
+  fs.writeFileSync(
+    path.resolve(dirPath, 'README.txt'),
+    `
+> Why do I have a folder named ".expo" in my project?
+
+The ".expo" folder is created when an Expo project is started using
+"expo start" command.
```

Comment:
we should get rid of the linebreak here and put the description on one line. this lets the text editor determine when to wrap lines